drivers/sensor/icm42688/icm42688.h

```diff
@@ -359,15 +359,13 @@ static inline void icm42688_temp_c(int32_t in, int32_t *out_c, uint32_t *out_uc)
 {
 	int64_t sensitivity = 13248; /* value equivalent for x100 1c */
 
-	int64_t in100 = in * 100;
+	/* Shift by 25 degrees Celsius */
+	int64_t in100 = in * 100 + 25 * sensitivity;
```

Comment:
Nit: Add parenthesis to help with order of operation.

Reply:
Done